### DISCUSSION TITLE
Where can I use the stats?

### DISCUSSION THREAD

**Discussion by ghost** [vichan-devel/vichan#624] [Ideas]
No longer in use

**Comment by perdedora** [comment]
What do you mean?

**Comment by ghost** [comment]


**Reply by perdedora** [reply]
You choose between the following themes: Recent and Basic. Recent is almost the same as 4chan homepage.
To install them, head over to: mod.php?/themes and configure them

**Comment by ghost** [comment]


**Reply by perdedora** [reply]
I don't know. Ask tinyib dev for support.
